.formatter.exs

```diff
@@ -15,5 +15,6 @@
     # Errors tests
     assert_eval_raise: 3
   ],
-  normalize_bitstring_modifiers: false
+  normalize_bitstring_modifiers: false,
+  charlists_as_sigils: false
```

Comment:
Don't we want it to be true?

Reply:
I disabled it because some tests relying on the charlist AST were breaking:
https://github.com/elixir-lang/elixir/pull/12064/commits/44e6b316f671eb5f5630c6a57ec93d48db753a8f

Maybe we can fix each of these tests in a different way (`Code.string_to_quoted`...) or just test the AST for the sigil (but this would be a different case). WDYT?

Reply:
I think all of those tests could be rewritten to charlists except for the ones in `quoted_ast_test`. However, I think the quoted_ast tests can actually be simplified. Instead of `quoted_to_string(quote do: 'foo')`, we could write: `quoted_to_string(~c"foo")` and that's going to be enough in all cases except the cases with interpolation. For the interpolation cases then it may be better to build the AST by hand?

```
iex(1)> quote do: 'foo#{:oops}'
{{:., [], [List, :to_charlist]}, [],
 [["foo", {{:., [], [Kernel, :to_string]}, [], [:oops]}]]}
```

Reply:
OK I will give it a shot. BTW, the ExUnit [diff](https://github.com/elixir-lang/elixir/commit/44e6b316f671eb5f5630c6a57ec93d48db753a8f#diff-07c6f937a496c3563c46ca6678532a9cb9dd55830807cc45f42449ff43a9e032R1014) feels like a regression, maybe it should handle the `~c` sigils as well?

```elixir
     code:  assert diff_left =~ expected_left
     left:  "-~c\"fox hops over 'the dog\"-"
     right: "'fox -ho-ps over -\\'-the -dog-'"
```

Reply:
Yes, it should it seems. :)